Blockzilla/Siri/SiriFavoriteViewController.swift

```diff
@@ -214,6 +214,8 @@ extension SiriFavoriteViewController: UITextFieldDelegate {
 extension SiriFavoriteViewController: INUIAddVoiceShortcutViewControllerDelegate {
     func addVoiceShortcutViewController(_ controller: INUIAddVoiceShortcutViewController, didFinishWith voiceShortcut: INVoiceShortcut?, error: Error?) {
         controller.dismiss(animated: true, completion: nil)
```

Comment:
Dismissing and popping in the same method doesn't look like a good solution. Can we determine somehow the way the controller is shown?

Reply:
Well, we have two controllers first one is INUIAddVoiceShortcutViewController the one that manages Siri Intent that is presented (using .formsheet style) and is first dismissed after creating the shortcut, and then we have our view controller where the user adds the url that wants to be opened using siri shortcut that is pushed on the navigation stack from settings and we should pop it after successfully creating Siri shortcut. This was the intended behaviour - we have a vc from system and our own vc - this is why we have dismissing and popping. I think the user functionality is fine like this even if we have dismissing and after that popping. What do you think?